cmd/integration/integration.go

```diff
@@ -63,9 +63,10 @@ import (
 	"k8s.io/kubernetes/test/integration"
 	"k8s.io/kubernetes/test/integration/framework"
 
-	"github.com/coreos/go-etcd/etcd"
+	etcd "github.com/coreos/etcd/client"
```

Comment:
Can you please add a comment to Watch() and WatchList() functions in:
pkg/storage/interface.go
about what resourceVersion semantics is (that we start right after the passed version).